engine/src/main/java/org/terasology/engine/modes/StateMainMenu.java

```diff
@@ -43,134 +44,140 @@
 import org.terasology.utilities.Assets;
 
 /**
- * The class implements the main game menu.
- * <br><br>
+ * The class implements the main game menu. <br>
+ * <br>
  *
  * @version 0.3
  */
 public class StateMainMenu implements GameState {
-    private Context context;
-    private EngineEntityManager entityManager;
-    private EventSystem eventSystem;
-    private ComponentSystemManager componentSystemManager;
-    private NUIManager nuiManager;
-    private InputSystem inputSystem;
+	private Context									context;
```

Comment:
Tabs instead of spaces and strange alignment. Just please use spaces :-) The GitHub diff here shows the alignment off anyway, rarely works so we don't use it. You may want to tweak your IDE config to match our code style.